src/System.Security.Cryptography.Algorithms/tests/HashAlgorithmTest.cs

```diff
@@ -40,17 +63,29 @@ protected void Verify(byte[] input, string output)
             {
                 Assert.True(hash.HashSize > 0);
                 actual = hash.ComputeHash(input, 0, input.Length);
-            }
 
-            Assert.Equal(expected, actual);
+                Assert.Equal(expected, actual);
+
+#if netstandard17
+                actual = hash.Hash;
+                Assert.Equal(expected, actual);
+#endif
+            }
         }
 
         protected void VerifyRepeating(string input, int repeatCount, string output)
         {
             using (Stream stream = new DataRepeatingStream(input, repeatCount))
             {
-                Verify(stream, output);
+                VerifyComputeHashStream(stream, output);
+            }
+
+#if netstandard17
+            using (Stream stream = new DataRepeatingStream(input, repeatCount))
+            {
+                VerifyICryptoTransformStream(stream, output);
             }
+#endif
```

Comment:
There are some quirky behaviors in NetFx HashAlgorithm that we should have tests to ensure we don't change unless we mean to.

* TransformBlock(a) followed by ComputeHash(b) is the same as ComputeHash(a.Concat(b))
* TransformBlock(a), TransformFinalBlock(b), TransformBlock(a), TransformFinalBlock(b) is (I think) ComputeHash(a.Concat(b).Concat(a).Concat(b))
* TransformBlock(a), TransformFinalBlock(b) => x, ComputeHash(Array.Empty\<byte\>() => same x => ComputeHash(Array.Empty\<byte\>()) => correct empty hash answer
* TransformBlock(a), TransformFinalBlock(b) => x, ComputeHash(Array.Empty\<byte\>() => same x => TransformBlock(a), TransformFinalBlock(b) => same x (because ComputeHash reset the state).

Also, a whole bunch of exception tests for TransformBlock and TransformFinalBlock.